crates/starknet-devnet/src/main.rs

```diff
@@ -198,20 +204,51 @@ async fn main() -> Result<(), anyhow::Error> {
 
     info!("Starknet Devnet listening on {}", address);
 
-    if starknet_config.dump_on == Some(DumpOn::Exit) {
-        server.with_graceful_shutdown(shutdown_signal(api.clone())).await?
-    } else {
-        server.await?
+    let empty_handle: task::JoinHandle<()> = task::spawn(async {});
+    let mut block_interval_handle = empty_handle;
+    if let BlockGeneration::Interval(timestamp) = starknet_config.block_generation_on {
```

Comment:
`timestamp` is a misleading name; timestamp is the exact time at which something happens. What we have is a time duration specifier. Something like `time` or `seconds` would be better IMO.

Reply:
fixed in f16183413cf643da5994b7e6b080c84942298f92